extension/lib/shopify/storefront-client/ServerError.js

```diff
@@ -0,0 +1,22 @@
+class ShopifyStorefrontClientServerError extends Error {
+  constructor (statusCode, url, requestBody) {
```

Comment:
please add jsdoc for all method + constructor

Reply:
done